### ISSUE TITLE
CI tests are failing on `master`

### ISSUE THREAD

**Issue by frewsxcv** [rust-fuzz/cargo-fuzz#323]
https://github.com/rust-fuzz/cargo-fuzz/actions/runs/3256280093/jobs/5346527710

**Comment by frewsxcv** [comment]
> so that other PRs can land in the mean time

yeah god forbid someone merge prs that have failing tests. who would do that??

**Comment by fitzgen** [comment]
heh, its mostly for new PRs that come in so we can easily differentiate between existing failing tests vs new failures easily when CI comes back failing

**Comment by Shnatsel** [comment]
On the latest master (commit ) the compiler [straight up panicked](https://github.com/rust-fuzz/cargo-fuzz/actions/runs/3275992832/jobs/5391512146#step:6:364) when building with Memory Sanitizer:

>  Error: internal compiler error: unexpected panic
> note: the compiler unexpectedly panicked. this is a bug.
>
> note: we would appreciate a bug report: https://github.com/rust-lang/rust/issues/new?labels=C-bug%2C+I-ICE%2C+T-compiler&template=ice.md
> 
> note: rustc 1.66.0-nightly (06f049a35 2022-10-17) running on x86_64-unknown-linux-gnu
> 
> note: compiler flags: --crate-type lib -C opt-level=3 -C embed-bitcode=no -Z force-unstable-if-unmarked -C passes=sancov-module -C llvm-args=-sanitizer-coverage-level=4 -C llvm-args=-sanitizer-coverage-inline-8bit-counters -C llvm-args=-sanitizer-coverage-pc-table -C llvm-args=-sanitizer-coverage-trace-compares -C link-dead-code -Z sanitizer=memory -Z sanitizer-memory-track-origins -C llvm-args=-sanitizer-coverage-stack-depth -C debug-assertions -C codegen-units=1

I guess we should file a bug report, as suggested?

**Comment by VTCAKAVSMoACE** [comment]
As a workaround, you can pass `--strip-dead-code`. See: https://github.com/rust-lang/rust/issues/101930

**Comment by VTCAKAVSMoACE** [comment]
Resolved?
